### ISSUE TITLE
[Rain FX] and [Single Pass Stereo] Rain tyres texture is rendered different for each eye in VR

### ISSUE THREAD

**Issue by ludwig6666** [ac-custom-shaders-patch/acc-extension-config#282]
If I turn Single Pass Stereo off I see the tyres correctly, If I put Single Pass Stereo On the left eye I see all the tyres textures shifted to left. with both eyes open you can obviously see a superposition of textures.

the enviroment:

- CSP 0.1.79-preview5
- Pure 0.102
- HP Reverb 2
- Nvidia 3080Ti
- Open Composite
- no FSR

![left eye](https://user-images.githubusercontent.com/88636572/173287643-cc144bc5-fb48-4ad4-b68c-0a8eb78fd6df.png)

**Comment by ludwig6666** [comment]
similar issue with falling rain drops

**Comment by maxsteep** [comment]
And sun/godrays do not render in the right eye with single stereo pass - maybe same issue? Latest build 023 too

**Comment by bazookaben** [comment]
Yeah this is still an issue. So far I've only noticed it on the tires of Van Diemens and Mygale f1600 mod cars

**Comment by ludwig6666** [comment]
still present in 0.2.4-preview1

**Comment by ludwig6666** [comment]
Bug  still present in CsP preview 0.2.6 preview 1 😭😭
